core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1250,21 +1250,23 @@ private void updateMediaSession(final PlayerStatus playerStatus) {
 
         UiModeManager uiModeManager = (UiModeManager) getApplicationContext()
                 .getSystemService(Context.UI_MODE_SERVICE);
-        if (uiModeManager.getCurrentModeType() == Configuration.UI_MODE_TYPE_CAR) {
-            sessionState.addCustomAction(
-                new PlaybackStateCompat.CustomAction.Builder(
-                        CUSTOM_ACTION_REWIND,
-                        getString(R.string.rewind_label), R.drawable.ic_notification_fast_rewind)
-                        .build());
+        // Hack to work around this bug https://github.com/AntennaPod/AntennaPod/issues/5481#issuecomment-999811393
+        if (Build.VERSION.SDK_INT >= 31 ||
+                uiModeManager.getCurrentModeType() == Configuration.UI_MODE_TYPE_CAR) {
             sessionState.addCustomAction(
                 new PlaybackStateCompat.CustomAction.Builder(
                         CUSTOM_ACTION_FAST_FORWARD,
                         getString(R.string.fast_forward_label), R.drawable.ic_notification_fast_forward)
                         .build());
+            sessionState.addCustomAction(
```

Comment:
It's odd that I needed to move the FAST_FORWARD and REWIND lines to have the rewind button first, follow by fast foward